MVend/gwiza-dashboard
# ft():add multiple file upload functionalities

#### Description of changes made
---

- Added multiple files upload form
- Added error checking 
- Added real-time progress check with the socket.io

#### How should this be manually tested?
---

> **Provide the steps to manually test the functionalities implemented in this PR**

- After cloning the [repo](https://github.com/MVend/gwiza-dashboard.git), and `cd [the project directory]`
- Check out this branch by `git checkout [the branch for this pr]`
- Run `npm install` to install the project dependencies
- Run `npm run start` to start the server in the development mode.


#### Checklist:


#### Related issue

#

#### Blocker
> **If there is any blocker you have just mention it below**

#### Screenshots

![image](https://user-images.githubusercontent.com/50277906/126367872-082ecd35-0ddd-4548-bf81-6373d3882dfa.png)
![image](https://user-images.githubusercontent.com/50277906/126368391-cccf9168-d5b0-40af-ac52-6db340a078f9.png)
